website/docs/r/hdinsight_hadoop_cluster.html.markdown

```diff
@@ -192,6 +192,10 @@ A `storage_account` block supports the following:
 
 -> **NOTE:** This can be obtained from the `id` of the `azurerm_storage_container` resource.
 
+* `storage_resource_id` - (Optional) The ID of the Storage Account. Changing this forces a new resource to be created.
+
+-> **NOTE:** The `storage_resource_id` takes format like "/subscriptions/$SUBSCRIPTION_ID/resourceGroups/$RESOURCE_GROUP/providers/Microsoft.Storage/storageAccounts/$ACCOUNT_NAME".
```

Comment:
we strongly recommend against hard-coding Resource ID's in configurations, users should instead use Data Sources to look these up, [such as in this case](https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/data-sources/storage_account)

```suggestion
```

Reply:
@tombuildsstuff thanks for letting me know that. Fixed.